float64[2] vw
float64[2] encod
float64[2] odo
    # float64[1] rpm
    # float64[1] diffv